src/renderer/components/Login-Styles.js

```diff
@@ -0,0 +1,221 @@
+      }
+    }
+  }
+  .bp3-form-group label.bp3-label {
+    padding-left: 10px;
+    color: ${StyleVariables.colors.deltaChatPrimaryFgLight};
+    font-size: 16px; 
+  }
+`
+
+export const DeltaSelect = React.memo((props) => {
+  const [ isFocused, setIsFocused ] = useState(false)
+
+  const onFocus = () => setIsFocused(true)
+  const onBlur = () => setIsFocused(false)
+
+  return (
+    <DeltaSelectWrapper>
+      <FormGroup>
+        <DeltaLabel focus={isFocused}>{props.label}</DeltaLabel>
+        <div className='bp3-select .modifier'>
+          <select
+            id={props.id}
+            value={props.value}
+            onChange={props.onChange}
+            onFocus={onFocus}
+            onBlur={onBlur}
+          >
+            {props.children}
+          </select>
+        </div>
+      </FormGroup>
+    </DeltaSelectWrapper>
+  )
+})
+
+export const DeltaInputWrapper = styled(DeltaFormGroup)`
+  .bp3-input {
+    padding: unset;
+    border-radius: unset;
+    -webkit-box-shadow: none;
+    box-shadow: none;
+    border-bottom: 2px solid;
+    border-bottom-color: ${StyleVariables.colors.deltaChatPrimaryFgLight};
+    font-size: 16px; 
+
+    &:focus {
+      border-bottom-color: ${StyleVariables.colors.deltaFocusBlue};
+      color: ${StyleVariables.colors.deltaFocusBlue};
+    }
+
+    &:focus::placeholder {
+      color: transparent;
+    }
+    &::placeholder {
+      color: ${StyleVariables.colors.deltaChatPrimaryFgLight};
+    }
+  }
+
+  .bp3-button.bp3-minimal.bp3-intent-warning, .bp3-button.bp3-minimal.bp3-intent-warning:hover {
+    color: #62656a !important;
+  }
+
+  .bp3-button.bp3-minimal.bp3-intent-warning:hover {
+    background-color: #d0d0d0 !important;
+  }
+`
+
+export const DeltaLabel = styled.div`
+    visibility: ${props => props.visible === false ? 'hidden' : 'visible'};
+    height: 13px;
+    font-size: 13px;
+    line-height: 13px;
+    width: 100%;
+    color: ${props => props.focus === true
+    ? StyleVariables.colors.deltaFocusBlue
+    : StyleVariables.colors.deltaChatPrimaryFgLight};
+`
+
+export const DeltaInput = React.memo((props) => {
+  const [ isFocused, setIsFocused ] = useState(false)
+
+  const onFocus = () => setIsFocused(true)
+  const onBlur = () => setIsFocused(false)
+
+  return (
+    <DeltaInputWrapper>
+      <FormGroup>
+        <DeltaLabel
+          visible={isFocused || props.value.length > 0}
+          focus={isFocused}
+        >{props.placeholder}
+        </DeltaLabel>
+        <InputGroup
+          id={props.id}
+          type={props.type}
+          value={props.value}
+          onChange={props.onChange}
+          placeholder={props.placeholder}
+          min={props.min}
+          max={props.max}
+          disabled={props.disabled}
+          onFocus={onFocus}
+          onBlur={onBlur}
+          rightElement={props.rightElement}
+        />
+      </FormGroup>
+    </DeltaInputWrapper>
+  )
+})
+
+export const DeltaPasswordInput = React.memo((props) => {
```

Comment:
What's this `.memo()` thingie?

Reply:
It's memoizing, so this component doesn't get rerendered/reexecuted if the props stay the same.

Reply:
Cool!